client/src/Models/Ringdown.js

```diff
@@ -144,11 +194,27 @@ class Ringdown {
   // Validators
 
   get isPatientValid() {
```

Comment:
I haven't looked at this in awhile, but now I'm wondering why we have both `isPatientValid()` and `validationData`, which we only look at for the required, non-range fields.  Seems like `validationData` should tell us if the form is valid, otherwise, why do we have it?  Which isn't due to this PR, but it feels like there's still something overly complicated going on with ringdown validation.

Reply:
To handle this in new ticket encapsulating error handling in new class.